ffs/integrationtest/integration_test.go

```diff
@@ -1389,6 +1421,10 @@ func requireJobState(t *testing.T, fapi *api.API, jid ffs.JobID, status ffs.JobS
 			require.True(t, ok)
 			require.Equal(t, jid, job.ID)
 			if job.Status == ffs.Queued || job.Status == ffs.Executing {
+				if job.Status == status {
+					stop = true
+					res = job
+				}
```

Comment:
Considering that now `requireJobState` might be called for _Executing_ scenario (instead of always final states), just stopping earlier if that's the case.